PDP10/ka10_dpy.c

```diff
@@ -243,8 +243,10 @@ t_stat dpy_devio(uint32 dev, uint64 *data) {
         clr_interrupt(dev);
         uptr->STAT_REG &= ~CONO_MASK;
         uptr->STAT_REG |= *data & CONO_MASK;
-        if (*data & CONO_INIT)
+        if (*data & CONO_INIT) {
             dpy_update_status( uptr, ty340_reset(&dpy_dev), 1);
+            ty340_set_grid(6, 9);

```

Comment:
I thought you had renamed this function ty342_set_grid?
Since (for now) the PDP-6/10, running MIT s/w is the only client, any reason not to make this the default?
OR if you keep the call in ka10_dpy, make a function that calls ty340_reset, sets the spacing and returns the 340 status??

Reply:
Yes, I renamed it in the SIMH pull request.  Thanks for the reminder, I should rename it here too.

The reason would be that the only documentation we have, for PDP-7, insists it's 7x12.  Maybe it was configured differently for the PDP-6, or MIT made a local modification.  I'm happy to make 6x9 the default; I think it's your call.